components/team/teamCard.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from "classnames";
+import Link from "next/link";
+
+import PilotCard from "@/components/pilot/pilotCard";
+import { components } from "@/types";
+
+interface Props {
+  team: components["schemas"]["Team"];
+}
+
+const TeamCard = ({ team }: Props) => {
+  const { _id, name, pilots } = team;
+  const urlName = name.toLowerCase().replace(/\s/g, "-");
+  return (
+    <div>
+      <figcaption className="px-2">
+        <h3>{name}</h3>
+      <PilotCard key={pilots[0].civlid} pilot={pilots[0]} />
```

Comment:
pilots[0] est juste un number (le civlid).
Le type de team.pilots est number[] au lieu de Pilot[]

Soit tu modifies l'API, soit on peut fetch les pilotes au front dans teams/index.tsx et balancer une prop pilots en plus de teams:
-fetch /pilots en plus de /teams

```tsx
{
  teams.map((team) => {
    const teamPilots = pilots.filter((pilot) =>
      team.pilots.includes(pilot.civlid)
    );
    return <TeamCard key={team._id} team={team} pilots={teamPilots} />;
  });
}
```

Reply:
fixed, wrong type defined (Team instead of TeamExport)